src/components/Sidebar/SidebarLinkItem.js

```diff
@@ -1,32 +1,50 @@
 import React, { useMemo } from 'react';
 import PropTypes from 'prop-types';
-import SidebarItem from './SidebarItem';
+import { SidebarItem } from './SidebarItem';
 import SidebarLink from './SidebarLink';
 import SidebarLinkMenu from './SidebarLinkMenu';
 import { EXPAND_MENU } from './expand-menu-enum';
+import { HighlightedMenuContext } from '../../contexts/highlighted-menu-context';
+
+const INDENTATION_INCREASE = 8;
 
 const SidebarLinkItem = ({
   uuid,
   label,
   link,
   content,
   level,
-  interactable = false,
+  indentOffset,
+  expandable = false,
   indent = 0,
   expandMenu = EXPAND_MENU.EXPANDED,
+  $active = false,
 }) => {
+  const nextIndent = indentOffset > 0 ? indent : indent + INDENTATION_INCREASE;
+  const nextIndentOffset = indentOffset - 1;
   const linkContent = useMemo(
     () =>
       Array.isArray(content) ? (
-        <SidebarLinkMenu key={label} data={content} indent={indent + 16} expandMenu={expandMenu} />
+        <HighlightedMenuContext.Consumer>
+          {(highlightedMenuId) => (
+            <SidebarLinkMenu
+              key={label}
+              data={content}
+              indent={nextIndent}
+              expandMenu={expandMenu}
+              indentOffset={nextIndentOffset}
+              highlightedMenuId={highlightedMenuId}
+            />
+          )}
+        </HighlightedMenuContext.Consumer>
       ) : (
-        <SidebarLink to={link} $leaf={true} indent={indent}>
+        <SidebarLink to={link} $leaf={true} $active={$active} indent={indent}>
```

Comment:
I may be unaware of some `docs` convention but is there a reason this prop is prefixed?

Reply:
@Glazy Sorry, I got distracted before writing this comment:
This is a Styled Component Transient Prop: https://styled-components.com/docs/api#transient-props
The dollar prefix prevents the prop from being passed to the underlying React component, while the styled component can still make use of the information.